StockWatch.Backend/src/main/java/com/henning/oden/java/StockAlert/Backend/entities/StockWatch.java

```diff
@@ -0,0 +1,72 @@
+package com.henning.oden.java.StockAlert.Backend.entities;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.*;
+
+import javax.persistence.*;
+import java.util.Objects;
+
+@Entity
+@Table(name = "stock_watches")
+@NoArgsConstructor
+@AllArgsConstructor
+@EqualsAndHashCode
+public class StockWatch {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Getter @Setter
+    private long id;
+
+    @Column
+    @Getter @Setter
+    private long userId;
+
+    @Column
+    @Getter @Setter
+    private long stockId;
+
+    @Column
+    @Getter @Setter
+    private double minPrice;
+
+    @Column
+    @Getter @Setter
```

Comment:
do we need such annotations on fields?

Reply:
Without them JPA seems to rely on calling the no arg constructor which would then leave the fields uninitialized when fetching entities of this type from the database.